staging/src/k8s.io/cloud-provider/config/scheme/scheme.go

```diff
@@ -20,8 +20,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	"k8s.io/cloud-provider/app/apis/config"
-	"k8s.io/cloud-provider/app/apis/config/v1alpha1"
+	"k8s.io/cloud-provider/config"
```

Comment:
This package should probably be called "install" instead?

Reply:
Was referring to `k8s.io/cloud-provider/config/scheme`

Reply:
Was just going to ask^^. Thanks for the further explain :)

Reply:
Then I might also change the func name to install instead of `AddToSchema`